dmoj/cptbox/tracer.py

```diff
@@ -30,29 +29,28 @@
 log = logging.getLogger('dmoj.cptbox')
 
 _PIPE_BUF = getattr(select, 'PIPE_BUF', 512)
-_SYSCALL_INDICIES: List[Optional[int]] = [None] * 7
-
-if 'freebsd' in sys.platform:
-    _SYSCALL_INDICIES[DEBUGGER_X64] = 4
-else:
-    _SYSCALL_INDICIES[DEBUGGER_X86] = 0
-    _SYSCALL_INDICIES[DEBUGGER_X86_ON_X64] = 0
-    _SYSCALL_INDICIES[DEBUGGER_X64] = 1
-    _SYSCALL_INDICIES[DEBUGGER_X32] = 2
-    _SYSCALL_INDICIES[DEBUGGER_ARM] = 3
-    _SYSCALL_INDICIES[DEBUGGER_ARM64] = 5
-
-# (python arch, executable arch) -> debugger
-_arch_map = {
-    (ARCH_X86, ARCH_X86): DEBUGGER_X86,
-    (ARCH_X64, ARCH_X64): DEBUGGER_X64,
-    (ARCH_X64, ARCH_X86): DEBUGGER_X86_ON_X64,
-    (ARCH_X64, ARCH_X32): DEBUGGER_X32,
-    (ARCH_X32, ARCH_X32): DEBUGGER_X32,
-    (ARCH_X32, ARCH_X86): DEBUGGER_X86_ON_X64,
-    (ARCH_ARM, ARCH_ARM): DEBUGGER_ARM,
-    (ARCH_A64, ARCH_ARM): DEBUGGER_ARM,
-    (ARCH_A64, ARCH_A64): DEBUGGER_ARM64,
+_SYSCALL_INDICIES: List[Optional[int]] = [None] * 5
+
+_SYSCALL_INDICIES[PTBOX_ABI_X86] = 0
+_SYSCALL_INDICIES[PTBOX_ABI_X64] = 1
+_SYSCALL_INDICIES[PTBOX_ABI_X32] = 2
+_SYSCALL_INDICIES[PTBOX_ABI_ARM] = 3
+_SYSCALL_INDICIES[PTBOX_ABI_ARM64] = 5
```

Comment:
Why skip 4?

Reply:
It's FreeBSD which is not actually supported as an ABI for now.